services/slam/builtin/builtin_test.go

```diff
@@ -952,12 +954,14 @@ func TestSLAMProcessSuccess(t *testing.T) {
 
 	createFakeSLAMLibraries()
 
+	mapRate := 200
+
 	attrCfg := &builtin.AttrConfig{
 		Algorithm:        "fake_orbslamv3",
 		Sensors:          []string{"good_color_camera"},
 		ConfigParams:     map[string]string{"mode": "mono", "test_param": "viam"},
 		DataDirectory:    name,
-		MapRateSec:       200,
+		MapRateSec:       &mapRate,
```

Comment:
Note: cant do &(200) or anything like that so I have had to define the mapRate variable to create the pointer. If anyone has a better way to do that let me know

Reply:
only thought would be making them constants(idk if that would work here) or just making a `good_mapRate` and `bad_mapRate`.